test/fixtures/0-dns/createCert.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+const fs = require('fs');
+const crypto = require('crypto');
+const rfc5280 = require('asn1.js-rfc5280');
+const asn1 = require('asn1.js');
+const BN = asn1.bignum;
+
+const id_at_commonName = [ 2, 5, 4, 3 ];
```

Comment:
inconsistent casing, sometimes `snake_case`, sometimes `camelCase`, it looks like test/fixtures doesn't get linted